mne/connectivity/envelope.py

```diff
@@ -0,0 +1,61 @@
+# Authors: Eric Larson <larson.eric.d@gmail.com>
+#          Sheraz Khan <sheraz@khansheraz.com>
+#          Denis Engemann <denis.engemann@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+
+from ..filter import next_fast_len
+
+
+def orthogonal_envelope_correlation(data):
```

Comment:
Can we call it `envelope_correlation` or so and make the orthoganalization an optional argument?
I'd then make a private orthogonalization function that we can enrich with more option, e.g., Housholder or QR decomposion for multivariate orthogonalization.

Reply:
I'll rename but let's save the refactoring for when we actually need it